f2/src/LocationPage.js

```diff
@@ -31,7 +31,9 @@ export const LocationPage = () => {
               <LocationInfoForm
                 onSubmit={data => {
                   dispatch({ type: 'saveFormData', data: { location: data } })
-                  history.push(doneForms ? '/confirmation' : '/contactinfo')
+                  history.push(
+                    doneForms ? '/confirmation' : '/locationAnonymous',
```

Comment:
I don't think we want to show both location+locationAnonymous pages. We only want to see one of them, depending on what the user chose on the anonymousInfo page.

Reply:
Yes, that is right. That's why i created issue #1686 to correct the flow. if I don't put them in the current flow. People can't see both of the pages.

Reply:
Ok I understand. I suggest we leave it out of the flow. People can access it by doing `npm run dev` and accessing the page directly. Then we'll fix the flow in #1686!!